src/components/MainContent.js

```diff
@@ -62,45 +70,188 @@ export default function MainContent() {
     );
   });
 
+  const tableData = [
+    {
+      logo: xdlogo,
+      project: "Material XD Version",
+      members: [
+        {
+          profile: smiley,
+          name: "Jane Simmons",
+          stackOrder: 0,
+          left: "left-[0px]",
+        },
+        {
+          profile: smiley,
+          name: "Jon Doe",
+          stackOrder: 1,
+          left: "left-[15px]",
+        },
+        {
+          profile: smiley,
+          name: "Rumi Hadid",
+          stackOrder: 2,
+          left: "left-[30px]",
+        },
+        {
+          profile: smiley,
+          name: "Jane Doe",
+          stackOrder: 3,
+          left: "left-[45px]",
+        },
+      ],
+      budget: "$14,000",
+      completion: 60,
+    },
+    {
+      logo: letterlogo,
+      project: "Add Progress Track",
+      members: [
+        {
+          profile: smiley,
+          name: "Rumi Hadid",
+          stackOrder: 0,
+          left: "left-[0px]",
+        },
+        {
+          profile: smiley,
+          name: "Jessica Doe",
+          stackOrder: 1,
+          left: "left-[15px]",
+        },
+      ],
+      budget: "$3,000",
+      completion: 10,
+    },
+    {
+      logo: slacklogo,
+      project: "Fix Platform Errors",
+      members: [
+        {
+          profile: smiley,
+          name: "Marissa Ro",
+          stackOrder: 0,
+          left: "left-[0px]",
+        },
+        {
+          profile: smiley,
+          name: "Jane Doe",
+          stackOrder: 1,
+          left: "left-[15px]",
+        },
+      ],
+      budget: "Not set",
+      completion: 100,
+    },
+    {
+      logo: spotifylogo,
+      project: "Launch our Mobile App",
+      members: [
+        {
+          profile: smiley,
+          name: "Jane Dems",
+          stackOrder: 0,
+          left: "left-[0px]",
```

Comment:
It is probably unnecessary to have the css related data here. When you're looping over it, you have access to things like index and can still make use of it if you needed it for things like z-index.

Reply:
without the css though, i think that all the profiles will be stacked on top of each other and you would only be able to see the first one on top..

Reply:
thanks for the clarification, made the necessary changes!